CHANGELOG.rst

```diff
@@ -18,6 +18,42 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 5.3.1 (2019-11-25)
+=========================
+
+No significant changes.
+
+
```

Comment:
This is "duplicated".
Would be better to do this after https://github.com/pytest-dev/pytest/pull/6272 also, which improves the existing snippets already.

Reply:
@nicoddemus 
Although #6272 is approved, I do not want to create a conflict here by merging it myself.
So please feel free to merge #6272 yourself if you think it is good either before, or after (but needs conflict resolution then - iff the improved changelogs make sense; I've merely done it to show what can be done there).

Reply:
Sure, I will fix the conflicts after merging this one. 👍

Reply:
(I will only be able to get to this much later though, so feel free to get at it if you want/can).